scripts/manual_testing/refactor/all.ts

```diff
@@ -0,0 +1,88 @@
+import {getRouterContractAddress, colors} from "./utils/utils";
+import { TransactionBuilder } from "./utils/TransactionBuilder";
+import {
+    generateUsers,
+    mint,
+    addLiquidity,
+    swap,
+    removeLiquidity
+} from "./index"
+
+// Get the arguments passed to the script
+const args = process.argv.slice(2);
+
+// Access the individual arguments
+/**
+ * Represents the network used for testing.
+ * @type {string}
+ */
+const network = args[0];
+
+const mode = args[1];
+
+console.log("network:", network)
+console.log("running on:", mode)
+let txMaker: TransactionBuilder;
+
+
+
+const testAll = async (network: string, mode: string) => {
+    let apiUri: string;
+    let rpcUri: string;
+    let routerContractAddress: string;
+    switch (network) {
+        case "standalone":
+            console.log(colors.green, 'Using standalone network')
+            apiUri = "http://soroswapCoreApi:8010"
+            rpcUri = "http://stellar"
+            routerContractAddress = await getRouterContractAddress(apiUri, network)
+
+            txMaker = new TransactionBuilder(
+                `${rpcUri}:8000`,
+                `${rpcUri}:8000/soroban/rpc`,
+                `${rpcUri}:8000/friendbot?addr=`,
+                routerContractAddress,
+                network
+            );
+        break;
+        case "testnet":
+            console.log(colors.green, 'Using testnet network')
+            apiUri = "https://horizon-testnet.stellar.org"
+            rpcUri = "http://stellar"
+            routerContractAddress = await getRouterContractAddress(apiUri, network)
+            txMaker = new TransactionBuilder(
+                `${rpcUri}`,
+                `${rpcUri}/soroban/rpc`,
+                "https://friendbot.stellar.org/?addr=",
+                routerContractAddress,
+                network
```

Comment:
You have duplicated code, 

move txMaker to after the switch statement (define variables inside switch, define txMaker only once after the switch)

Reply:
Fixed!